src/routes/add_user.js

```diff
@@ -0,0 +1,21 @@
+const MongoClient = require('mongodb').MongoClient;
+const dbQueries = require('../db_queries.js');
+const bcrypt = require('bcrypt');
+
+module.exports = (req, res) => {
+  MongoClient.connect(process.env.DATABASE_URL, (err, db) => {
```

Comment:
This doesn't seem like a great pattern.
Probably better to have a file where you will store the basic queries (eg `db_queries.js`), and at the top of that file connect to the db. Also require that file in `start.js` so that connection is opened when server starts.
From that file you can export the queries and use them elsewhere without connecting afresh to MongoClient each time.